app/validators/email_address_validator.rb

```diff
@@ -10,7 +10,7 @@ class EmailAddressValidator < ActiveModel::EachValidator
 
   def validate_each(record, attribute, value)
     if value.blank? || !value.match?(EMAIL_REGEX)
-      record.errors[attribute] << I18n.t('activerecord.errors.models.candidate.attributes.email_address.invalid')
```

Comment:
Can we remove this string as well?

Reply:
Yes, definitely. Have removed repeated occurrences of it as well now that we validate all email fields with the custom validator.